PluginClasses/Zee5PluggablePlayer.swift

```diff
@@ -453,4 +453,8 @@ public class Zee5PluggablePlayer: APPlugablePlayerBase, ZPAdapterProtocol {
         
         return result
     }
+    
+    func addGesturePanDownHandler(_ handler: @escaping () -> ()) {
+        ZEE5PlayerManager.sharedInstance().setPanDownGestureHandler(handler);
```

Comment:
why it's needed to add the method here?

Reply:
the HybridScreen can call it directly, no?

Reply:
I rechecked, you are right.
Done